app/models/requisition.rb

```diff
@@ -1,4 +1,4 @@
-class ApplicantsPosition < ApplicationRecord
+class Requisition < ApplicationRecord
   belongs_to :applicant
   belongs_to :position
   enum status: [:applicant_interested, :currently_interviewing, :intend_to_hire]
```

Comment:
Which one should be the "I do not want to hire" choice? Separately, 'applicant_interested' is indicated by the relationship itself, right? So we don't need that?